src/TriboroughBridge-ChorusPlugin/CommandLineProcessor.cs

```diff
@@ -84,7 +84,7 @@ public static class CommandLineProcessor
 							//This is an unparsable command line
 							Console.WriteLine(CommonResources.kInvalidCommandLineOptions);
 							//Signal FLEx or other apps
-							throw new ApplicationException(String.Format("Invalid command line options: {0}.", args));
+							throw new ApplicationException(String.Format("Invalid command line options: {0}. Please launch from FLEx.", args));
```

Comment:
Strictly speaking, FB was designed to be run by any number of clients, with FLEx being one of them. So, it would be better to not tell users to run FB from FLEx. (Yes, the design may not have been implemented in all cases with multiple clients in mind, but that was the design.)